### DISCUSSION TITLE
[QUESTION] symsync_create_kaiser() design

### DISCUSSION THREAD

**Discussion by pfeatherstone** [jgaeddert/liquid-dsp#384] [General]
Can you explain the `fc` and `As` parameters in the following? I don't understand the choices.

https://github.com/jgaeddert/liquid-dsp/blob/9f32eb7b956738ac18d6d19ac0a7da5ea6e1e5a0/src/filter/src/symsync.proto.c#L232-L236

**Comment by tom-huntington** [comment]
https://dsp.stackexchange.com/a/38565
